Sources/Imperial/Helpers/Request+Imperial.swift

```diff
@@ -10,29 +10,29 @@ extension Request {
     ///   - service: The service to get the data from.
     /// - Returns: An instance of the type passed in.
     /// - Throws: Errors from trying to get the access token from the request.
-    func create<Model: FederatedCreatable>(_ model: Model.Type, with service: OAuthService)throws -> Future<Model> {
-        let uri = try service[model.serviceKey].value(or: ServiceError.noServiceEndpoint(model.serviceKey))
-        
-        let token = try service.tokenPrefix + self.accessToken()
-        
-        return try self.make(Client.self).get(uri, headers: ["Authorization": token]).flatMap(to: Model.self, { (response) -> Future<Model> in
-            return try model.create(from: response)
-        }).map(to: Model.self, { (instance) -> Model in
-            let session = try self.session()
-            try session.set("imperial-\(model)", to: instance)
-            return instance
-        })
+    func create<Model: FederatedCreatable>(_ model: Model.Type, with service: OAuthService, on req: Request) throws -> EventLoopFuture<Model> {
+//        let url = try service[model.serviceKey].value(or: ServiceError.noServiceEndpoint(model.serviceKey))
+//
+//        let token = try service.tokenPrefix + req
+//            .accessToken()
+//
+//        return self.make(Client.self).get(URI(string: url), headers: ["Authorization": token]).flatMap { response in
+//            return try! model.create(from: response).map { instance in
+//                try session.set("imperial-\(model)", to: instance)
+//                return instance
+//            }
+//        }
+        fatalError()
```

Comment:
This needs to be implemented I'm guessing?

Reply:
done